sections/preface.tex

```diff
@@ -0,0 +1,50 @@
+\chapter*{Preface\markboth{Preface}{Preface}}\label{ch:preface}
+\addcontentsline{toc}{chapter}{Preface}
+
+\vspace{\baselineskip}\hfill Aalborg Universitet, May 28, 2018
```

Comment:
Er det her noget der bliver skrevet? For så er 2018 lidt et blwo up

Reply:
I fix good job well played nicely done